sdk/helper/certutil/helpers.go

```diff
@@ -9,16 +9,24 @@ import (
 	"crypto/rsa"
 	"crypto/sha1"
 	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
 	"encoding/pem"
 	"errors"
 	"fmt"
 	"math/big"
+	"net"
+	"net/url"
 	"strconv"
 	"strings"
+	"time"
 
+	"github.com/hashicorp/errwrap"
 	"github.com/hashicorp/vault/sdk/helper/errutil"
 	"github.com/hashicorp/vault/sdk/helper/jsonutil"
 	"github.com/mitchellh/mapstructure"
+	"golang.org/x/crypto/cryptobyte"
+	asn12 "golang.org/x/crypto/cryptobyte/asn1"
```

Comment:
I don't really like `asn12` here. It makes it seem like it's an implementation of some other version of ASN.1, e.g. ASN.1.2. Not sure what you found wrong with the name before, but please change this.

Reply:
Done.  It wasn't deliberate, I think my IDE chose that name during a refactoring operation when it saw the conflict with encoding/asn1.